src/core/qfieldcloud/qfieldcloudprojectsmodel.cpp

```diff
@@ -699,7 +700,7 @@ void QFieldCloudProjectsModel::loadProjects( const QJsonArray &remoteProjects, b
     QVariantHash projectDetails = project.toObject().toVariantHash();
     QFieldCloudProject *cloudProject = QFieldCloudProject::fromDetails( projectDetails, mCloudConnection );
 
-    if ( cloudProject->name() == QStringLiteral( "localized_datasets" ) )
+    if ( cloudProject->isSharedDatasetsProject() )
```

Comment:
Much nicer, thanks for adding the detail on QFC.

Reply:
Should have been there since the start, sorry for the delay.